client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,147 @@ const InstructionsRenderer = ({
         commandsContent
     );
 
-    const assertions = [...pageContent.instructions.assertions.assertions];
-    const assertionsContent = parseListContent(assertions);
-
+    headingLevel = 5;
     const Heading = `h${headingLevel}`;
 
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
-            <Heading>{pageContent.instructions.assertions.header}</Heading>
-            {pageContent.instructions.assertions.description}
-            <NumberedList>{assertionsContent}</NumberedList>
             {settingsContent.length ? settingsContent : null}
 
+            {renderableContent.commands.map(
+                ({ id, settings, assertionExceptions = [] }, i) => {
+                    const settingsScreenText = isV2
+                        ? renderableContent.target.at.raw.settings[settings]
+                              ?.screenText ?? ''
```

Comment:
```suggestion
                              ?.screenText ?? null
```

Because line 220 expects this to be null. Otherwise, settings show me shown as 'COMMAND ()' instead of just 'COMMAND'.

The following screenshot is from Command Button Example Test Plan V23.12.13 > Test 1 > VoiceOver. Not all VoiceOver commands may have a setting applied can lead to the empty brackets.

<img width="888" alt="screenshot showing Control+Option+Right Arrow and Tab commands with empty open and close brackets beside it" src="https://github.com/w3c/aria-at-app/assets/7191577/0f78cc17-1b7e-4d77-a9bf-dd8cd2a09bbc">